Python/test/bonds.py

```diff
@@ -20,6 +20,7 @@
 
 class FixedRateBondTest(unittest.TestCase):
     def setUp(self):
+        QuantLib.Settings.instance().setEvaluationDate(QuantLib.Date(2,1,2010))
```

Comment:
This needs to be rolled back by

    QuantLib.Settings.instance().setEvaluationDate(QuantLib.Date())

in the `tearDown` method, otherwise the tests that follow will find this particular evaluation date set, too (which might not be what they expect).